### ISSUE TITLE
Download page delivers potentially malicious software

### ISSUE THREAD

**Issue by TJEvans** [tourdedave/the-internet#53]
Tests of our automation framework ended up downloading the steam_api.dll yesterday and my network security team got alerted.  I believe this is a false positive, but to prevent future headaches please remove the offending files.

**Comment by wilson-mar** [comment]
@ TJEvans it's been over a year now. Is this still an issue?

**Comment by TJEvans** [comment]
Yes, this is an issue your team should take seriously.  This app allows arbitrary uploads and serves them back to other page visitors.
